tools/topology/CMakeLists.txt

```diff
@@ -96,6 +96,9 @@ set(TPLGS
 	"sof-apl-asrc-pcm512x\;sof-apl-asrc-pcm512x"
 	"sof-apl-src-pcm512x\;sof-apl-src-pcm512x"
 	"sof-cml-rt5682\;sof-cml-rt5682\;-DPLATFORM=cml"
+	"sof-cml-rt5682\;sof-cml-eq-fir-rt5682\;-DPLATFORM=cml\;-DHSMICPROC=eq-fir-volume"
+	"sof-cml-rt5682\;sof-cml-eq-fir-loud-rt5682\;-DPLATFORM=cml\;-DHSMICPROC=eq-fir-volume\;-DPIPELINE_FILTER1=eq_fir_coef_loudness.m4"
+	"sof-cml-rt5682\;sof-cml-eq-iir-rt5682\;-DPLATFORM=cml\;-DHSEARPROC=eq-iir-volume"
```

Comment:
The xPROC and PIPELINE_FILTERx look nice in the makefile.

Seems like the PR with new filter blobs is merged so we can use them.

Just need to think what examples to make. I'd set up HSMICPROC in every topology with eq-iir-volume with 40 Hz 0 dB high-pass. The high-pass is needed to prevent HDA codec headset glitches. Since the EQ in the capture does permanent impact the demo responses work better for playback.

A demo topology could be made with additional loudness IIR for headset. Also the speaker could get the eq-volume (should be renamed to eq-iir-eq-fir-volume for consistency) playback pipeline with IIR setup with pass and FIR set with pass response. I upstreamed the pass-trough blobs those switch the EQs to bypass. It consumes even less resources than flat.

Reply:
Thank you for your comments. PIPELINE_FILTER1 to xPROC_FILTERy changes and handling from middle layer need extra effort to address. I will create a new issue and define scope of the work. 

The other comments are very reasonable and minor work to do. I can address them today.